2020 09 30 question (#10432)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/accesspackagelocalizedcontent.md

---
title: "accessPackageLocalizedContent resource type"

> description: "**TODO: Add Description**"

Reviewer: update needed

author: "**TODO: Provide Github Name. See [topic-level metadata reference](https://msgo.azurewebsites.net/add/document/guidelines/metadata.html#topic-level-metadata)**"
localization_priority: Normal

> ms.prod: "**TODO: Add MS prod. See [topic-level metadata reference](https://msgo.azurewebsites.net/add/document/guidelines/metadata.html#topic-level-metadata)**"

Reviewer: update needed

doc_type: resourcePageType
---

# accessPackageLocalizedContent resource type

Namespace: microsoft.graph

A complex type used to represent a text in multiple localalized forms. It includes a default text, which is used in any case where the requested localization is not available.

## Properties

> |Property|Type|Description|Required|Read-only|

Reviewer 2: ```suggestion
|Property|Type|Description|
```

> |:---|:---|:---|:---|:---|

Reviewer 2: ```suggestion
|:---|:---|:---|
```

> |defaultText|String|**TODO: Add Description**|

Reviewer: I think this article needs some edits.

> |defaultText|String|The fallback string, which is used when a requested localization is not available.|Yes|No|

Reviewer 2: ```suggestion
|defaultText|String|The fallback string, which is used when a requested localization is not available. Required.
```

> |localizedTexts|[accessPackageLocalizedText](../resources/accesspackagelocalizedtext.md) collection|Content represented in a format for a specific locale.|No|No|

Reviewer 2: ```suggestion
|localizedTexts|[accessPackageLocalizedText](../resources/accesspackagelocalizedtext.md) collection|Content represented in a format for a specific locale. |
```

It is assumed that a property is not required unless explicitly stated.
